internal/app/orchestrator/downstream.go

```diff
@@ -110,12 +110,7 @@ func (d *downstreamResponseMap) getAggregatedKeys(m *mapper.Mapper) (map[string]
 func (responseChannel *responseChannel) addResponse(resp transport.Response) error {
 	responseChannel.wg.Add(1)
 	defer responseChannel.wg.Done()
-	ok, err := responseChannel.watch.Send(resp)
-	if err != nil {
-		return err
-	}
```

Comment:
What's the reason for removing the `err` return here?

Reply:
The Send function earlier used casting and returned an error when cast failed
```
func (w *watchV2) Send(s Response) (bool, error) {
	resp, ok := s.(*ResponseV2)
	if !ok {
		return false, fmt.Errorf("payload %s could not be casted to DiscoveryResponse", s)
	}
```

With the addition of VersionedRequest/Response, we got rid of casting, and there's no more error being returned by Send.
```
func (w *watchV2) Send(s Response) bool {
	select {
	case w.out <- gcpv2.PassthroughResponse{DiscoveryResponse: s.Get().V2, Request: *s.GetRequest().V2}:
		return true
	default:
		return false
	}
}
```